From 2ae92bda1fb881460aa44c33b1386ba8cdb355eb Mon Sep 17 00:00:00 2001
From: hanzongcheng <hanzongcheng@huawei.com>
Date: Sat, 25 Mar 2023 13:04:08 +0800
Subject: [PATCH] mica_main: set phy_shared_mem to 0x90000000

* The reserved memory was reprogrammed according to the memory
  layout of the hi3093, reserving 0x90000000 to 0x95000000.
  So update phy_shared_mem.

Signed-off-by: hanzongcheng <hanzongcheng@huawei.com>

diff --git a/mica_demo/mica_main.c b/mica_demo/mica_main.c
index 8f99d20..a82cec3 100644
--- a/mica_demo/mica_main.c
+++ b/mica_demo/mica_main.c
@@ -9,7 +9,7 @@
 
 static struct client_os_inst client_os = {
     /* physical address start of shared device mem */
-    .phy_shared_mem = 0x70000000,
+    .phy_shared_mem = 0x90000000,
     /* size of shared device mem */
     .shared_mem_size = 0x30000,
     .vring_size = VRING_SIZE,
-- 
2.34.1

